prime-router/src/main/kotlin/history/azure/DatabaseDeliveryAccess.kt

```diff
@@ -141,4 +144,47 @@ class DatabaseDeliveryAccess(
             query.fetchInto(DeliveryFacility::class.java)
         }
     }
+
+    /**
+     * Get a list of matching delivery facilities for a set of [reportIds]
+     * @link https://github.com/CDCgov/prime-reportstream/issues/9272
+     */
+    fun fetchBulkDeliveryFacilities(
```

Comment:
See https://kotlinlang.org/docs/kotlin-doc.html for improving kdoc.

Reply:
Fixed up the kdoc